doc/mediasdk-man.md

```diff
@@ -2187,17 +2187,25 @@ This function initiates execution of an asynchronous function not already starte
 | | |
 --- | ---
 `MFX_ERR_NONE` | The function completed successfully.
+`MFX_ERR_NONE_PARTIAL_OUTPUT` | The function completed successfully, bitstream contains a portion of the encoded frame according to required granularity.
 `MFX_WRN_IN_EXECUTION` | The specified asynchronous function is in execution.
 `MFX_ERR_ABORTED` | The specified asynchronous function aborted due to data dependency on a previous asynchronous function that did not complete.
 
 **Change History**
 
 This function is available since SDK API 1.0.
+Return code MFX_ERR_NONE_PARTIAL_OUTPUT is introduced in SDK API TBD.
 
 **Remarks**
 
 See status codes for specific asynchronous functions.
 
+In partial output mode, enabled during encoder initialization [mfxExtPartialBitstreamParam](#mfxExtPartialBitstreamParam) or reset, application is expected to call SyncOperation with the same sync point repeatedly until it receives the MFX_ERR_NONE or some error;
+when MFX_ERR_NONE_PARTIAL_OUTPUT code is returned, new portion of the encoded frame is appended to the last bitstream portion located at DataOffset and increment a DataLength size accoding to new portion size in the associated [mfxBitstream](#mfxBitstream) buffer.
+Finally, when MFX_ERR_NONE returned the bitstream buffer contains full bitstream for encoded frame with DataLengh equal to it length. 
+Any processing on app level of the partial result returned in bitstream buffer must be finished before next SyncOperation call.
```

Comment:
Just to double check: we guarantee that mediasdk will change mfxBitstream description (DataLength at the first place) only within SyncOperation call?

Reply:
yes, it works in the same way currently for frame based encoding, no difference for partial output here